README.md

```diff
@@ -117,6 +117,8 @@ For syntax, see [lookup.json](./lib/lookup.json), the available attributes are:
 "script": /path/to/script | https://url/to/script - Use a custom test script
 "sha": "<git-commit-sha>"    Test against a specific commit
 "envVar"                     Pass an environment variable before running
+"verify-node-gyp-called": true - Asserts that npm called node-gyp with either 'build' or 'rebuild'
+"vertify-node-gyp-not-called": true - Asserts that npm did not call node-gyp
```

Comment:
s/vertify/verify. Also in what situations do we need to verify that node-gyp wasn't called? I understand making sure it was called for checking v8 ABI compatibility, but I can't think of a use for the opposite.